src/base/fsparse.jl

```diff
@@ -20,10 +20,10 @@ See also: [`fiber!`](@ref)
 
 ```jldoctest
 julia> println(summary(fiber(sparse([1 0; 0 1]))))
-2×2 Fiber!(Dense(SparseList(Element(0))))
+2×2 Fiber!(Dense(SparseList(Element(0, Int64))))
```

Comment:
Can position type default to Int64? I'd rather not have to think about position type every time I make a level

Reply:
It does. That is actually the index type and it defaults to `Int`

Reply:
But do you mean in the printing of the types?

Reply:
yeah. I think that whenever we print an Element level, we should check if the postype or indextype is equal to the default before printing them out. Most users will never use these features